utils/src/main/kotlin/Utils.kt

```diff
@@ -275,6 +275,16 @@ fun Exception.collectMessages(): List<String> {
     return messages
 }
 
+/**
+ * Assume that the String contains a file path and replace all occurences of [File.separatorChar] with '/'.
+ */
+fun String.normalizePath() = replace(File.separatorChar, '/')
```

Comment:
Why not also call this "normalize*d*Path()" for consistency?

Reply:
I decided to use extension properties instead of functions, because:
* Both File and String are immutable, so the API looks better to me.
* The operation is cheap.
* Then the name "normalizedPath" which I also prefer is valid, because functions should use verbs.